src/classes/class-cachemanager.php

```diff
@@ -114,6 +120,20 @@ public function __construct() {
 			}
 		}
 
+		/**
+		 * Set static class for the Beaver Builder plugin (FlBuilderModel).
+		 */
+		public function setFlbuilder( \FLBuilderModel $fl_builder ) {
+			$this->flbuilder = $fl_builder;
```

Comment:
How do you set this? It does not look like it's being set at all...

Reply:
They are set automatically when the class property is called.

Reply:
Right, I'm not seeing that.

Reply:
@dz0ny I added the missing setter methods in the class. Also, having `object` type in the function args is safe to use. PHP does not throw an error if the object type does not exist (unless we call the setter method in the absence of the beaver builder plugin which we are not doing)